way_to_home/user_profile/urls.py

```diff
@@ -2,9 +2,11 @@
 
 from django.urls import path
 
-from .views import UserProfileView
+from user_profile.views import UserProfileView, put_access_token, update_telegram_id
 
 
 urlpatterns = [
     path('', UserProfileView.as_view(), name='profile'),
+    path('/telegram_redis', put_access_token, name='telegram_redis'),
```

Comment:
It is better to change it to `telegram_access_token`.
Because URL it is a public thing and it is not good to show where we store `access token` xD

Reply:
done